website/views/shared/profiles/achievements.jade

```diff
@@ -164,21 +164,21 @@ div(ng-if='profile.achievements.quests || user._id == profile._id')
         td x{{v}}
   hr
 
-div(ng-if='profile.achievements.snowball')
+div(ng-if='::profile.achievements.snowball')
   .achievement.achievement-snowball
   h5=env.t('annoyingFriends')
   small
     =env.t('annoyingFriendsText', {snowballs: "{{profile.achievements.snowball}}"})
```

Comment:
You can add the one time bindings to `snowballs: "{{::profile.achievements.snowball}}"` too.